README.md

```diff
@@ -1,5 +1,7 @@
 # MemoryCache
 
+[![CI](https://github.com/kostya/memory_cache/workflows/CI/badge.svg)](https://github.com/kostya/memory_cache/actions?query=workflow%3ACI)
```

Comment:
The CI has not run yet, that's why the badge does not exist. It will in the future.